File: backend/src/AtmosAQ.Application/LatestData/Queries/LatestDto.cs
using System.Collections.Generic;
using AtmosAQ.Application.Shared;

namespace AtmosAQ.Application.LatestData.Queries
{
    public class LatestDto
    {
        public string Location { get; set; }

        public string City { get; set; }

        public string Country { get; set; }

        public CoordinateDto Coordinates { get; set; }
        
        public List<MeasurementDto> Measurements { get; set; }
    }
}

File: frontend/src/app/@core/models/latest-dto.ts
import { CoordinateDto } from "./coordinate-dto";
import { MeasurementDto } from "./measurement-dto";

export interface LatestDto {
    location?: string | undefined;
    city?: string | undefined;
    country?: string | undefined;
    coordinates?: CoordinateDto;
    measurements?: MeasurementDto[] | undefined;
}